setup.cfg

```
@@ -8,7 +8,9 @@ author = Gustavo Niemeyer
 author_email = gustavo@niemeyer.net
 maintainer = Paul Ganssle
 maintainer_email = dateutil@python.org
-url = https://dateutil.readthedocs.io
+url = https://github.com/dateutil/dateutil
+project_urls =
+    Documentation = https://dateutil.readthedocs.io/en/stable/
```

Would be good to include an explicit link to the source as well, for programmatic processing of metadata:

```suggestion
    Documentation = https://dateutil.readthedocs.io/en/stable/
    Source = https://github.com/dateutil/dateutil
```